endpoint/BUILD

```diff
@@ -38,9 +38,9 @@ cc_binary(
         "endpoint_core.h",
     ],
     defines = [
-        "TA_HOST=\"localhost\"",
-        "TA_PORT=\"443\"",
-        "TA_SSL_SEED=\"nonce\"",
+        "ENDPOINT_HOST=\"node.deviceproof.org\"",
+        "ENDPOINT_PORT=\"5566\"",
+        "ENDPOINT_SSL_SEED=\"nonce\"",
```

Comment:
Maybe these should not be changed in this pull request.
Instead, I should be in charge of these change in the future pull request related to https://github.com/DLTcollab/tangle-accelerator/issues/671?

Reply:
But I think it is ok to change the naming from `TA` to `ENDPOINT`.